.github/workflows/build-image.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 Catena-X
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: build
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - v1.*
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+         
+      - name: Set up JDK 18
+        uses: actions/setup-java@v3
+        with:
+          java-version: '18'
+          distribution: 'temurin'
+        
+      - name: Build
+        run: mvn -B package
+        
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract Metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=semver,pattern={{version}},value=v1.7.0
```

Comment:
Please let the action automatically create the semver tags. Do not use hard coded versions in action. Otherwise you have to change each time the workflow.

Here you can find a example how to setup semver tagging: [TRG 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-1#implementation)

Reply:
Image tagging is still using hard coded tags (in this case 1.7.1), for file `.github/workflows/build.yml` it was fixed. Please fix also this one, or is it really on purpose to explicit name _1.7.1_ here?